src/cljs/rems/language_switcher.cljs

```diff
@@ -18,13 +18,9 @@
           (for [language languages]
             (let [lang-str (str/upper-case (name language))]
               [:form.inline
-               [:a {:href "#"
-                    :on-click (fn [event]
-                                (.preventDefault event)
-                                (rf/dispatch [:set-current-language language]))
-                    :class (lang-link-classes current-language language)
-                    :style {:padding 0}}
-                lang-str]])))))
+               #_(anti-forgery-field)
```

Comment:
where did this come from?

Reply:
It has been there since the beginning, 34e440ff. I had removed it in this PR, but then I reverted my changes to that file, so the mystery line lives on.